So there's a plague on/sheet.json

```diff
@@ -0,0 +1,8 @@
+
+{
+	"html": "So_theres_a_plague.html",
+	"css": "So_theres_a_plague.css",
+	"authors": "Indigo",
+	"preview": "So_theres_a_plague.png",
+	"instructions": "?????"

```

Comment:
```suggestion
	"instructions": "A character sheet for 'So, there is a plague on' by Indigo. Rules available [here](https://4b0082.itch.io/buboes)"
```

Reply:
I'd suggest to include the at least this much info in this field, leaving question marks just makes it confusing. Look at any existing Roll20 sheet for comparison.